packaging/package.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+	echo "
+Usage: $(basename "$0") [-h] [-m dir] [-b nbr] [-d dist] [-a arch] [-c commit_id] [-v version]
+ -- Generate debian package from fog_sw module.
+Params:
+    -h  Show help text.
+    -m  Module directory to generate deb package from (MANDATORY).
+    -b  Build number. This will be tha last digit of version string (x.x.N).
+    -d  Distribution string in debian changelog.
+    -c  Commit id of the git repository HEAD
+    -s  Subdirectory to be packaging
+    -g  Git version string
+	-v  Package version
+"
+	exit 0
+}
+
+check_arg() {
+	if [ "$(echo $1 | cut -c1)" = "-" ]; then
+		return 1
+	else
+		return 0
+	fi
+}
+
+error_arg() {
+	echo "$0: option requires an argument -- $1"
+	usage
+}
+
+mod_dir=""
+build_nbr=0
+distr=""
+version=""
+git_commit_hash=""
+git_version_string=""
+
+while getopts "hm:b:d:c:g:v:" opt
+do
+	case $opt in
+		h)
+			usage
+			;;
+		m)
+			check_arg $OPTARG && mod_dir=$OPTARG || error_arg $opt
+			;;
+		b)
+			check_arg $OPTARG && build_nbr=$OPTARG || error_arg $opt
+			;;
+		d)
+			check_arg $OPTARG && distr=$OPTARG || error_arg $opt
+			;;
+		c)
+			check_arg $OPTARG && git_commit_hash=$OPTARG || error_arg $opt
```

Comment:
not necessary to pass git_commit_hash as a argument, because this script could resolve it too

Reply:
Done